src/contextmenu.js

```diff
@@ -42,6 +46,10 @@ function getVscodeLink({
 
             if (line) {
                 vscodeLink += `:${line}:1`;
+            } else if (remoteHost != '') {
```

Comment:
```suggestion
            } else {
```

I wonder whether we could just simply always default to the first line to keep the code simpler? (I would leave your new comment in any case just to make sure we don't remove this logic)